ledger-tool/src/main.rs

```diff
@@ -2291,7 +2291,8 @@ fn main() {
         .subcommand(
             SubCommand::with_name("latest-optimistic-slots")
                 .about("Output up to the most recent <num-slots> optimistic \
-                        slots with their hashes and timestamps.")
+                        slots with their hashes and timestamps. By default, \
+                        exclude slots with no non-vote transactions")
```

Comment:
nit:
s/exclude slots with no non-vote transactions/exclude slots which contain only vote transactions/
Removes the double negative and is more consistent with the help message for `include-vote-only-slots`.